packages/jupyter/package.json

```diff
@@ -49,5 +49,8 @@
     "@types/react-dom": "^16.8 || ^17.0 || ^18.0",
     "react": "^16.8 || ^17.0 || ^18.0",
     "react-dom": "^16.8 || ^17.0 || ^18.0"
+  },
+  "devDependencies": {
+    "vitest": "^0.34.6"
```

Comment:
This is defined in the package and shouldn't need to be pulled in here.

Reply:
linter was complaining at some point will revert